### PR TITLE
Create ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57098](https://cumulocity.atlassian.net/browse/MTM-57098)
Original PR: [5765](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5765)

### CODE REVIEW

**Review by janhommes — APPROVED** [review]

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57098
+version: 1019.7.13
+---
+The availability feature has been disabled for the home dashboard. This change prevents issues with incorrect availability status being shown on the home dashboard.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md:17]:
```suggestion
The availability feature has been disabled for the Home dashboard. This change prevents issues with an incorrect availability status being shown on the Home dashboard.
```

File: content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Home dashboard availability disabled
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md:3]:
```suggestion
title: Home dashboard availability disabled
```

Comment (BeateRixen) [reply content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md:3]: @janhommes This sounds misleading. The availability of the Home dashboard has not been disabled. Is it "Availability status for Home dashboard disabled"?

Comment (eniosultan) [reply content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md:3]: @BeateRixen  The availability "input" when editing a dashboard was removed from home dashboards...
this is how it looks when it is included (in non-home dashboards):
image

Comment (BeateRixen) [reply content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md:3]: And what means disabled from a user perspective? That it is no longer shown? That would mean, from a user perspective, that it has been removed.

Comment (eniosultan) [inline content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md:3]:
```suggestion
title: Dashboard availability setting has been removed from the Home dashboard. 
```

File: content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57098
+version: 1019.7.13
+---
+The availability feature has been disabled for the Home dashboard. This change prevents issues with an incorrect availability status being shown on the Home dashboard.
```
Comment (eniosultan) [inline content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md:17]:
```suggestion
The availability feature has been removed for the Home dashboard. This change prevents issues with an incorrect availability status being shown on the Home dashboard.
```

File: content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Dashboard availability setting has been removed from the Home dashboard.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md:3]:
```suggestion
title: Dashboard availability setting removed from the Home dashboard
```

File: content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57098
+version: 1019.7.13
+---
+The availability feature has been removed for the Home dashboard. This change prevents issues with an incorrect availability status being shown on the Home dashboard.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-7-13-dashboard-availability-is-now-disabled-for-the-home-dashboard-graft-release.md:17]:
```suggestion
The availability setting has been removed for the Home dashboard. This change prevents issues with an incorrect availability status being shown on the Home dashboard.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**